mobile/lib/main.dart

```diff
@@ -57,139 +59,110 @@ class MainPage extends StatelessWidget {
+                                    color: context.colors.highlightGreen,
+                                    height: 1.5,
+                                  ),
+                                ),
+                              ],
                             ),
                           ),
                         ),
-                        Padding(
-                          padding: const EdgeInsets.symmetric(
-                            horizontal: 2,
-                          ),
-                          child: Container(
-                            width: 8,
-                            height: 8,
-                            decoration: const BoxDecoration(
-                              color: Colors.grey,
-                              shape: BoxShape.circle,
-                            ),
-                          ),
+                      ),
+                      child: Container(
+                        padding: const EdgeInsets.all(80),
+                        child: Text(
+                          // TODO: Place for the spinning vinyl record
+                          '',
+                          style: context.textStyles.textSmall,
                         ),
-                      ],
-                    ),
-                  ),
-                  child: Container(
-                    padding: const EdgeInsets.all(20),
-                    child: Text(
-                      'Lorem ipsum',
-                      style: context.textStyles.textSmall,
+                      ),
                     ),
                   ),
-                ),
-              ),
-              Padding(
-                padding: _widgetPadding,
-                child: Row(
-                  children: [
-                    Expanded(
-                      child: ColorShadowedCard(
-                        shadowColor: context.colors.highlightPurple,
-                        header: Padding(
-                          padding: const EdgeInsets.symmetric(
-                            vertical: 2,
-                          ),
-                          child: Text(
-                            'Nagłówek',
-                            style: context.textStyles.textMedium,
-                          ),
-                        ),
-                        child: Container(
-                          padding: const EdgeInsets.all(20),
-                          child: Text(
-                            'Lorem ipsum',
-                            style: context.textStyles.textSmall,
+                  Padding(
+                    padding: _widgetPadding,
+                    child: Row(
+                      children: [
+                        Expanded(
+                          child: ColorShadowedCard2(
+                            shadowColor: context.colors.highlightGreen,
+                            header: Padding(
+                              padding: const EdgeInsets.all(3),
```

Comment:
Ewww, that's not a multiple of 2
```suggestion
                              padding: const EdgeInsets.all(4),
```